Singer.API/Services/UserService.cs

```diff
@@ -40,7 +40,7 @@ protected UserService(ApplicationDbContext context, IMapper mapper, UserManager<
             UserName = dto.Email
          };
 
-         var userCreationResult = await UserManager.CreateAsync(baseUser);
+         var userCreationResult = await UserManager.CreateAsync(baseUser, "Testpassword123!");
```

Comment:
Please add `//TODO` that this should be replaced with random password once we have an email service in place which can send out random passwords.
Alternatively, you could go the extra mile and add a random generated password now already, which is logged to the console or something so it can be retrieved from the logs.